toolkit/tools/imagecustomizerapi/mountidentifiertype.go

```diff
@@ -18,6 +18,8 @@ const (
 	// MountIdentifierTypePartLabel mounts this partition via the GPT PARTLABEL
 	MountIdentifierTypePartLabel MountIdentifierType = "part-label"
 
+	MountIdentifierTypeDeviceMapper MountIdentifierType = "device-mapper"
```

Comment:
The `mountidentifiertype` is a public API and can not be modified, use `ExtendedMountIdentifierTypeDev` of internal type instead.

Reply:
I guess the added type for dev-mapper is for modifying verity protection, if so, then the current main supports it, but correct me if I am wrong.